proposals/0029-cooperative-vector.md

```diff
@@ -102,16 +104,23 @@ void ps_main(args) // args: texture, normal, position
 
     // layer0 = inputVector*inputMatrix + biasVector0
     // The matrix and bias are loaded from memory at offsets : moffset0 and boffset0
-    vector<uint32_t, K> layer0 = MatrixVectorMulAdd(inputVector, inputMatrix0, moffset0, biasVector0, boffset0);
+    MatrixRef<DATA_TYPE_UINT32, K, M, MATRIX_LAYOUT_MUL_OPTIMAL> M0 = { inputMatrix0, moffset0, 0 }; 
```

Comment:
According to linalg.h, the template lists M before K. I think M should swap order with K.

Reply:
I've gone for more explicit INPUT_SIZE and OUTPUT_SIZE with the internal size being N.